src/test/resources/datasource/administrator/administrator_datasource.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM order_associations;
+DELETE FROM orders;
+
+INSERT INTO orders VALUES('1', '1660893861', '1660893864', 'Waiting for confirm', '0', '0', '1', '31', '2');
+INSERT INTO order_associations VALUES('1', '4', '13', '1');
```

Comment:
Maybe we have to remove only order that we created, not the whole order list?

Reply:
@dmatsev what is the best practice to work with data source in tests?

Reply:
The best option is to have parametrized queries, meaning you should pass values into the query. And also, it is better to use stored procedures instead of queries.
But for now, it's fine as is. The goal was to implement a class that works with databases, and it is done.